css/selectors/-moz-focus-inner.json

```
@@ -12,7 +12,8 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "≤72"
+              "version_added": "≤72",
```

While we're at it, this appears to have been supported since Firefox 1: https://github.com/caugner/firefox-releases/blame/d10ff74a78afb46fdb9f3662a0a7082197b3a127/layout/style/nsCSSPseudoElementList.h#L73

```suggestion
              "version_added": "1",
```